src/index.d.ts

```diff
@@ -0,0 +1,3 @@
+import GeoDaWasm from "./geoda-proxy";
+
+export function New(): Promise<GeoDaWasm>;
```

Comment:
Manually added index.d.ts to expose New() function, which is the only entry of the jsgeoda library, and return `GeoDaWasm` type.